fl_client/main.py

```diff
@@ -64,6 +64,10 @@ def init_fl_client_app(client: Client) -> None:
     for folder in ["request", "running"]:
         fl_client_folder = fl_client / folder
         fl_client_folder.mkdir(parents=True, exist_ok=True)
+        # Create a dummy file in the running folder
+        if folder == "running":
+            dummy_file = fl_client_folder / "dummy"
+            dummy_file.touch()
```

Comment:
I created a dummy file here so the running folder would get synced after the app is installed (since now we check if the app is installed by the running folder, but seems like it does not get synced. Is it because dummy will not be synced?